main/read.c

```diff
@@ -296,6 +296,39 @@ static boolean parseLineDirective (void)
  *   Input file I/O operations
  */
 
+#define MAX_IN_MEMORY_FILE_SIZE (1024*1024)
+
+static MIO *getMio (const char *const fileName, const char *const openMode)
+{
+	char buf[4096];
+	MIO *src, *dst;
+	size_t size = 0;
+
+	src = mio_new_file (fileName, openMode);
+	dst = mio_new_memory (NULL, 0, eRealloc, free);
```

Comment:
Oh, sorry, I took a memorandum here. I fixed this in

https://github.com/universal-ctags/ctags/commit/231e40e72126d0c32201fc72ec24a690bb784bc9

Sorry for making noise.